src/storage/src/hummock/store/version.rs

```diff
@@ -480,6 +487,14 @@ impl HummockVersionReader {
                         )
                         .await?
                         {
+                            if local_stats.bloom_filter_check_counts
```

Comment:
Will this cause multiple `inc()` for one key? Assume `sstable_infos.len() == 10`, and we get a bloom filter check positive in the first `sstable_info`. And since we do not reset `local_stats` in `local_stats.report()`. `bloom_filter_check_counts` will increase 10 tims.